### PR TITLE
Login link in navigation

### PR DESCRIPTION
**Pull request by anuschka**
Addressed issue #176.

We need to work on the link to member's profile and to edit the member's profile but first we somehow need to associate UserProfile with Github login information.

### CODE REVIEW

File: web/templates/base.html [hunk 1]
```diff
@@ -34,9 +34,36 @@
...
+					<ul class="nav navbar-nav navbar-right">
+                    <li class="dropdown" id="menuLogin">
+                        {% if user.is_authenticated %}
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                Hi, {{ user.first_name }}
+                                <b class="caret"></b>
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a href="#">Edit profile</a></li>
+                                <li><a href="#">Another action</a></li>
```
Comment (ialja) [inline web/templates/base.html]: How about a link to your profile? :) Also, please remove the "Something else" stuff ...

File: web/templates/base.html [hunk 2]
```diff
@@ -34,9 +34,36 @@
 						<li class="{% current 'pages-tasks' %}"><a href="{% url 'pages-tasks' %}">Tasks</a></li>
 						<li class="{% current 'pages-members' %}"><a href="{% url 'pages-members' %}">Members</a></li>
 						<li class="{% current 'pages-events' %}"><a href="{% url 'pages-events' %}">Events</a></li>
-						<li class="{% current 'pages-login' %}"><a href="{% url 'pages-login' %}">Login</a></li>
 						<li class="{% current 'pages-links' %}"><a href="{% url 'pages-links' %}">Help</a></li>
 					</ul>
+					<ul class="nav navbar-nav navbar-right">
+                    <li class="dropdown" id="menuLogin">
```
Comment (ialja) [inline web/templates/base.html]: Indentation is a bit all over the place ...

**Comment by ialja** [comment]
Still no link to the user's own profile? :)

File: web/views.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from api.models import Task
 
 def index(request):
-	return render_to_response('pages/index.html',{})
+	return render_to_response('pages/index.html',{},context_instance=RequestContext(request))
```
Comment (gandalfar) [inline web/views.py]: you don't need this line if you use `render()` as other functions do

**Comment by anuschka** [comment]
@gandalfar Without `,context_instance=RequestContext(request)` it would not render "hi, user.first_name" after signing in on the index page.

**Comment by gandalfar** [comment]
@anuschka `render` is a shortcut function for `render_to_response` with RequestContext, see 2. paragraph at: https://docs.djangoproject.com/en/1.7/topics/http/shortcuts/#render

**Comment by anuschka** [comment]
OK now I get it :smile: Thank you :chocolate_bar:

**Pull request merged**